CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.4 (2019-06-07)
+
+- Fixed the modal component to have correct accessibility tab controls. [#46](https://github.com/blackbaud/skyux-modals/pull/46)
```

Comment:
Fixed the modal component to correct accessibility tab controls.

Reply:
Also, might want to add a shout out to Jack.